Doc/library/logging.rst

```diff
@@ -544,6 +544,9 @@ The useful mapping keys in a :class:`LogRecord` are given in the section on
    .. versionchanged:: 3.2
       The *style* parameter was added.
 
+   .. versionchanged:: 3.8
+      Incorrect or mismatched style and fmt will raise a ``ValueError``. For
```

Comment:
Insert "The *validate* parameter was added." before "Incorrect or mismatched style..."